doc/sr_integration.rst

```diff
@@ -0,0 +1,211 @@
+*****************************
+Integrating into a Simulation
+*****************************
+
+========
+Overview
+========
+
+This document provides some general guidelines to integrate the SmartRedis client
+into existing simulation codebase. Developers of these simulation codebases will
```

Comment:
Change to "integrate the SmartRedis client into existing simulation codebases"

Reply:
Thank you; fixed.